ansible/roles/nginx/templates/site.j2

```diff
@@ -2,11 +2,11 @@
 # {{ ansible_managed }}
 #
 
-{% if item.server.get('use_balancer', False) == True %}
+{% if item.server.get('use_balancer', False) %}
 upstream {{ item.server.file_name }} {
   least_conn;
-{% for host in groups['webworkers'] %}
-  server {{ host }}:{{ django_port }};
+{% for host in groups[item.server.use_balancer] %}
```

Comment:
it's a bad name and will eventually change, but `use_balancer` used to be a boolean. now it's a string that denotes what set of servers you'd like to balance